index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="style.css">
+
+  <title>Project-X</title>
+</head>
+<body>
+  <header>
+    <div class="notif-bar">
+      <a href=".introduction"><h4>God-Logic</h4></a>
```

Comment:
- [ ] You did a great job 😃 at adding the required changes by the previous reviewer, but I see that the path of the page in the href attribute is not correct, so as stated by the previous reviewer put the path which navigates you to the home page when you click on it 
- take a look at this hint below ⤵️😊
```snippet
<a href="index.html"><h4>God-Logic</h4></a>

```

Reply:
Thank you for the hint I actually didn't know how to do that.